src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -2,6 +2,7 @@ Microsoft.CodeAnalysis.CodeStyle.NotificationOption.Severity.get -> Microsoft.Co
 Microsoft.CodeAnalysis.CodeStyle.NotificationOption.Severity.set -> void
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.AliasImportDeclaration(string aliasIdentifierName, Microsoft.CodeAnalysis.SyntaxNode name) -> Microsoft.CodeAnalysis.SyntaxNode
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.NameExpression(Microsoft.CodeAnalysis.INamespaceOrTypeSymbol namespaceOrTypeSymbol) -> Microsoft.CodeAnalysis.SyntaxNode
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.StringEscapeCharacter = "string - escape character" -> string
```

Comment:
>const [](start = 0, length = 5)

I see this is similar to other classifications like `VerbatimStringLiteral`. But I didn't understand why those names need to be part of the public API...

Reply:
Classifications can have orderings applied to them.  So, if someone wants to add their own classifications, but have them before/after roslyn classifications, they can use these names to do so.